Tools/Audit-Sentinel-Detection/Readme.md

```diff
@@ -0,0 +1,80 @@
+
+
+# Audit Sentinel Detection Rules
+
+### Prerequistes : [Configure Audit in Microsoft Sentinel](https://learn.microsoft.com/en-us/azure/sentinel/enable-monitoring)
```

Comment:
Hey @samikroy, please fix the typo Prerequistes to Prerequisites

Reply:
Fixed